python/src/migrate.py

```diff
@@ -0,0 +1,28 @@
+import os
+from db import config
+from db.client import ScyllaClient
+
+arg_parser = config.argument_parser()
+db_config = vars(arg_parser.parse_args())
+client = ScyllaClient(db_config)
```

Comment:
With the implementation that I suggested this will have a better usability since the user will not need to input their credentials every time.

Reply:
agree